DataFormats/MuonReco/interface/Muon.h

```diff
@@ -183,6 +183,43 @@ namespace reco {
 			   RPCHitAndTrackArbitration, GEMSegmentAndTrackArbitration, ME0SegmentAndTrackArbitration };
     
     ///
+    /// ====================== STANDARD SELECTORS ===========================
+    ///
+    enum Selector {
```

Comment:
IDSelector may be more descriptive

Reply:
It's not ID. It's any selector including isolation.